docs/FAQ.rst

```diff
@@ -0,0 +1,276 @@
+Q&A
+***
+
+Q: How do I know if DALI can help me?
+#####################################
+A: You need to check our docs first and see if DALI operators cover your use case. Then try to run
+a couple of iterations of your training with a fixed data source - a random tensor that is generated
```

Comment:
A random tensor doesn't necessarily suffice. How about suggesting loading one batch of data and reusing it in subsequent iterations?

Reply:
Fixed